ClearKeep/Scenes/GroupCall/Views/InComingVoiceGroupCall.swift

```diff
@@ -0,0 +1,179 @@
+
+	var statusCalling: some View {
+		Text("CallGroup.Incomming.Voice".localized)
+			.padding(.all)
+			.font(AppTheme.shared.fontSet.font(style: .input2))
+			.foregroundColor(foregroundColorGrey5)
+			.frame(maxWidth: .infinity, alignment: .center)
+	}
+
+	var userName: some View {
+		Text("UI Designs".localized)
+			.frame(maxWidth: .infinity, alignment: .center)
+			.font(AppTheme.shared.fontSet.font(style: .display2))
+			.foregroundColor(foregroundColorWhite)
+			.frame(maxWidth: .infinity, maxHeight: .infinity, alignment: .top)
+	}
+
+	var buttonResponse: some View {
+		HStack {
+			VStack {
+				Button {
+					isTappedDecline.toggle()
+				} label: {
+					ZStack {
+						Circle()
+							.strokeBorder(backgroundDecline, lineWidth: Constant.borderLineWidth)
+							.background(Circle().foregroundColor(backgroundDecline))
+							.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+						AppTheme.shared.imageSet.phoneOffIcon
+							.renderingMode(.template)
+							.foregroundColor(foregroundColorWhite)
+					}
+			}
+				Text("Call.Decline".localized)
+					.font(AppTheme.shared.fontSet.font(style: .body2))
+					.foregroundColor(foregroundColorWhite)
+			}
+			.frame(maxWidth: .infinity)
+
+			VStack {
+				Button {
+					isTappedAnswer.toggle()
+				} label: {
+					ZStack {
+						Circle()
+							.strokeBorder(backgroundAnswer, lineWidth: Constant.borderLineWidth)
+							.background(Circle().foregroundColor(backgroundAnswer))
+							.frame(width: Constant.paddingButtonNext, height: Constant.paddingButtonNext)
+						AppTheme.shared.imageSet.phoneCallIcon
+							.renderingMode(.template)
+							.foregroundColor(foregroundColorWhite)
+					}
+			}
+				Text("Call.Decline".localized)
+					.font(AppTheme.shared.fontSet.font(style: .body2))
+					.foregroundColor(foregroundColorWhite)
+			}
+			.frame(maxWidth: .infinity)
+		}
+		.frame(maxWidth: .infinity)
+	}
+}
+
+// MARK: - Color func
+private extension InComingVoiceGroupCall {
+	var foregroundColorWhite: Color {
+		AppTheme.shared.colorSet.offWhite
+	}
+
+	var foregroundColorBlack: Color {
+		AppTheme.shared.colorSet.black
+	}
+
+	var foregroundColorPrimary: Color {
+		AppTheme.shared.colorSet.primaryDefault
+	}
+
+	var foregroundColorGrey5: Color {
+		AppTheme.shared.colorSet.grey5
+	}
+
+	var foregroundColorGreyLight: Color {
+		AppTheme.shared.colorSet.greyLight
+	}
+
+	var foregroundCrossIcon: Color {
+		colorScheme == .light ? foregroundColorBlack : foregroundColorGreyLight
+	}
+
+	var backgroundGradientPrimary: LinearGradient {
+		LinearGradient(gradient: Gradient(colors: AppTheme.shared.colorSet.gradientPrimary), startPoint: .leading, endPoint: .trailing)
+	}
+
+	var backgroundDecline: Color {
+		AppTheme.shared.colorSet.errorDefault
+	}
+
+	var backgroundAnswer: Color {
+		AppTheme.shared.colorSet.successDefault
+	}
+}
+struct InComingVoiceGroupCall_Previews: PreviewProvider {
+	static var previews: some View {
+        InComingVoiceGroupCall()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/ClearKeep/ck-ios/blob/e7188fca22737d356920695ce8dd453fe3bcc8d0/ClearKeep/Scenes/GroupCall/Views/InComingVoiceGroupCall.swift#L177"></span>Please use tabs instead of spaces<br />
<code>not_spaces</code> <code>InComingVoiceGroupCall.swift:177</code></td>
    </tr>
  </tbody>
</table>